esmvaltool/config-developer.yml

```diff
@@ -16,8 +16,16 @@
 #
 # Only the default drs is mandatory, the others are optional
 ###############################################################################
+PRIMAVERA:
+  input_dir: '[expid]/original_files/cmorfiles/CMIP/[institute]/[model]/[exp]/[ensemble]/[mip]/[var]/[grid]/[version]'
+  input_file: '[var]_[mip]_[model]_[exp]_[ensemble]_[grid]_*.nc'
+  output_file: '[project]_[model]_[mip]_[exp]_[ensemble]_[field]_[var]_[start_year]-[end_year]'
+  cmor_type: 'CMIP6'
+  cmor_tables: '~/PycharmProjects/earthdiagnostics/earthdiagnostics/cmor_tables/primavera'
+
 
 CMIP5:
+  cmor_type: 'CMIP5'
```

Comment:
This needs to be also added to other projects based on the CMIP5-CMOR tables.
I get an error when running `namelist_perfmetrics_CMIP5.yml`

```
  File "/home/righ_ma/ESMValTool/public/esmvaltool/main.py", line 259, in run
    main()
  File "/home/righ_ma/ESMValTool/public/esmvaltool/main.py", line 199, in main
    process_namelist(namelist_file=namelist_file, config_user=cfg)
  File "/home/righ_ma/ESMValTool/public/esmvaltool/main.py", line 235, in process_namelist
    namelist = read_namelist_file(namelist_file, config_user)
  File "/home/righ_ma/ESMValTool/public/esmvaltool/namelist.py", line 38, in read_namelist_file
    raw_namelist, config_user, initialize_tasks, namelist_file=filename)
  File "/home/righ_ma/ESMValTool/public/esmvaltool/namelist.py", line 436, in __init__
    raw_namelist['diagnostics'])
  File "/home/righ_ma/ESMValTool/public/esmvaltool/namelist.py", line 453, in _initialize_diagnostics
    name, raw_diagnostic.get('variables'), models)
  File "/home/righ_ma/ESMValTool/public/esmvaltool/namelist.py", line 517, in _initialize_variable_collection
    self._initialize_variables(raw_variable, models)
  File "/home/righ_ma/ESMValTool/public/esmvaltool/namelist.py", line 482, in _initialize_variables
    _add_cmor_info(variable, cmor_keys)
  File "/home/righ_ma/ESMValTool/public/esmvaltool/namelist.py", line 196, in _add_cmor_info
    elif variable['cmor_table'] in CMOR_TABLES:
KeyError: 'cmor_table'
```